plugins/custom_commands.py

```diff
@@ -245,7 +248,8 @@ class CustomCommands(BasePlugin):
                 self.logger.exception("Exception occurred in custom command: ", exc_info=True)
                 await respond(msg, "**WARNING: An error occurred while running the custom command.**")
             else:
-                await respond(msg, res)
+                if res:
+                    await respond(msg, res)
```

Comment:
I'd rather you raise a syntax error here since either the CC or the input is invalid